### PR TITLE
Merge multi entry file to reduce msi size

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description
Before:
image

After:
image

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: scripts/installer/windows/scripts/main.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import sys
```
Comment (wangchao1230) [inline scripts/installer/windows/scripts/main.py:1]: add some PR description on the size we saved with this PR

File: scripts/installer/windows/scripts/pf.cmd [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@echo off
+setlocal
+
+set MAIN_EXE=%~dp0.\main.exe
```
Comment (wangchao1230) [inline scripts/installer/windows/scripts/pf.cmd:4]: maybe give this exe some other name more identifiable. better with pf prefix. e.g. pfcli?

File: scripts/installer/windows/scripts/main.py (hunk 1 above)
Comment (wangchao1230) [inline scripts/installer/windows/scripts/main.py:1]: add docstring to this file explain why we need this

File: scripts/installer/windows/scripts/pf.cmd [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@echo off
```
Comment (wangchao1230) [inline scripts/installer/windows/scripts/pf.cmd:1]: use .cmd or .bat?

File: scripts/installer/windows/scripts/pfs.cmd [hunk 4]
```diff
@@ -0,0 +1,5 @@
+@echo off
+setlocal
```
Comment (wangchao1230) [inline scripts/installer/windows/scripts/pfs.cmd:2]: why didn't we change pfsvc?

File: scripts/installer/windows/scripts/pf.cmd (hunk 2 above)
Comment (YingChen1996) [reply scripts/installer/windows/scripts/pf.cmd:4]: change main to pfcli

File: scripts/installer/windows/scripts/pf.cmd (hunk 3 above)
Comment (YingChen1996) [reply scripts/installer/windows/scripts/pf.cmd:1]: update to bat

File: scripts/installer/windows/scripts/pfs.cmd (hunk 4 above)
Comment (YingChen1996) [reply scripts/installer/windows/scripts/pfs.cmd:2]: updated

**Review by lalala123123 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**